tests/test_app.py

```diff
@@ -243,3 +244,44 @@ def test_find_by_ext_n(self) -> None:
             with open(json_filename, "r") as json_file:
                 report = json.load(json_file)
                 assert len(report) == 0
+
+    def test_zip_p(self) -> None:
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            samples_dir_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "samples")
+            json_filename = os.path.join(tmp_dir, "dummy.json")
+            # depth = 3
+            proc = subprocess.Popen([
+                sys.executable, "-m", "credsweeper", "--log", "silence", "--path", samples_dir_path, "--save-json",
+                json_filename, "--max_depth", "3"
+            ],
+                                    stdout=subprocess.PIPE,
+                                    stderr=subprocess.PIPE)
+            _stdout, _stderr = proc.communicate()
+            assert os.path.exists(json_filename)
+            with open(json_filename, "r") as json_file:
+                report = json.load(json_file)
+                assert len(report) == SAMPLES_POST_CRED_COUNT + 3
+            # depth = 1
+            proc = subprocess.Popen([
+                sys.executable, "-m", "credsweeper", "--log", "silence", "--path", samples_dir_path, "--save-json",
+                json_filename, "--max_depth", "1"
+            ],
+                                    stdout=subprocess.PIPE,
+                                    stderr=subprocess.PIPE)
+            _stdout, _stderr = proc.communicate()
+            assert os.path.exists(json_filename)
+            with open(json_filename, "r") as json_file:
+                report = json.load(json_file)
+                assert len(report) == SAMPLES_POST_CRED_COUNT + 1
+
+    def test_patch_multifile_p(self) -> None:
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        target_path = os.path.join(dir_path, "samples", "multifile.patch")
+        proc = subprocess.Popen([sys.executable, "-m", "credsweeper", "--diff_path", target_path, "--log", "silence"],
+                                stdout=subprocess.PIPE,
+                                stderr=subprocess.PIPE)
+        _stdout, _stderr = proc.communicate()
+        output = " ".join(_stdout.decode("UTF-8").split())
+        print(output)
+        assert output == 'returning from dumb path returning from dumb path returning from dumb path returning from dumb path'
```

Comment:
I checked this mysterious string.. and find out it comes from `whatthepatch`. (whatthepatch>patch.py#L248)
I think it is not good to add stdout TC that comes from other library.. not ours..
So how about remove those assert case?

Reply:
I'll move the test of ```whatthepath``` lib to separated test for the case - ok?

Reply:
Okay that's good idea.